DEVDOCS.md

```diff
@@ -0,0 +1,60 @@
+#Encore Link Developer Documentation
+EncoreLink is a web application currently built with (from back to front):
+
+* [PostgreSQL Database](https://www.postgresql.org/)
+* [Loopback API Framework](https://loopback.io/)
+* [Node.js Express Web Framework](http://expressjs.com/)
+* [Redux State Container](http://redux.js.org/)
+* [React.js UI Library](https://facebook.github.io/react/)
+* [ES6 JavaScript](http://es6-features.org) with [Babel JS compiler](https://babeljs.io/)
+* [Foundation by Zurb Front End Framework](http://foundation.zurb.com/)
```

Comment:
Yeah, I used Front End Framework because that's what Zurb called it. I agree CSS framework is probably more accurate. I'll make that change.